Dockerfile

```diff
@@ -5,10 +5,10 @@ WORKDIR /app
 COPY package.json package.json
 COPY package-lock.json package-lock.json
 
-RUN npm i --only=production
+RUN npm i
 
-COPY ./src ./src
```

Comment:
Why do you need to copy any other file ?

Reply:
I don't know. I got an error saying "Cannot use import statement outside a module", it disapeared if I copied everything.

Reply:
It might be the .babelrc not being copied causing that.